src/main/java/Car.java

```diff
@@ -0,0 +1,26 @@
+//자동차 이름, 위치, 자동차가 이동 했는지 안 했는지 등
+public class Car {
+    private final String name;
+    private int position = 0;
+    private final int MIN_GO_NUM = 4;
```

Comment:
```suggestion
    private static final int MIN_GO_NUM = 4;
```
MIN_GO_NUM은 final 변수 보다 고정된 값이므로 상수가 적절하지 않을까요??